### PR TITLE
Add userinfo FAQ

### PR DESCRIPTION
**Pull request by rebekaburnett**

### CODE REVIEW

**Comment by rebekaburnett** [comment]
@ErlendGramGjesdal @jonkjenn This is the proposal. I can move things back to Login API FAQ or have a variant. Please compare to: https://github.com/vippsas/vipps-login-api/pull/225

File: userinfo-api-faq.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+[Vipps MobilePay Technical Documentation](https://vippsas.github.io/vipps-developer-docs/docs/APIs/userinfo-api).
+
+<!-- END_COMMENT -->
+
+## Which scopes can I use? Why do I get `Invalid_scope`?
+
+If you get “Invalid_scope” this means that you have included one or more scopes
+that you do not have access to or that is not supported.
+See:
+[Scopes](api-guide/core-concepts.md#scopes).
```
Comment (jonkjenn) [inline userinfo-api-faq.md:34]: invalid_scope is a very login specific error code

File: userinfo-api-faq.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+* name
+* phoneNumber
+
+**Important:** You should never ask for more scopes than you need for your
+application. The user will need to consent to sharing the information with you
+so adding more scopes increases the chance that they will decline.
+
+The scope `openid` is required and does not require user consent. It
+provides the claim `sub` which is a unique id for the end user at that
+particular merchant.
```
Comment (jonkjenn) [inline userinfo-api-faq.md:51]: This is only relevant for login

File: userinfo-api-faq.md (hunk 1 above)
Comment (rebekaburnett) [reply userinfo-api-faq.md:34]: I will move it back to login.

File: userinfo-api-faq.md (hunk 2 above)
Comment (rebekaburnett) [reply userinfo-api-faq.md:51]: I will move it back to login api

File: userinfo-api-faq.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+[guidance from Datatilsynet](https://www.datatilsynet.no/rettigheter-og-plikter/personopplysninger/fodselsnummer/).
+
+According to the above NIN can only be used when it is legally required or when
+there is a need for a secure identification of the person and the NIN is
+required to achieve this. This applies e.g. when a company is required to
+report to the tax authorities or within healthcare and credit.
+
+Keeping a unique and consistent identifier for the user over time is not a
+sufficient requirement. For this purpose Vipps offers a unique merchant specific
+user identifier that are delivered as part of the registration/login. This is
```
[the remark this answers is not in the document]
Comment (rebekaburnett) [reply userinfo-api-faq.md:108]: I will move it back to login

File: userinfo-api-faq.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+the claim `sub` that is delivered based on the `openid` scope. This unique
+identifier will allow you to keep a consistent user profile even if the user
+changes contact information.
+
+Beware that Vipps Login it not an electronic ID. Thus, the NIN can only be
+used to simplify the customers processes by removing manual input or to look up
+the customer in your own or external registers. This can be done as part of the
+processes to become a customer or to link login with Vipps to an existing user.
```
[the remark this answers is not in the document]
Comment (rebekaburnett) [reply userinfo-api-faq.md:116]: I will move it back to login

**Comment by rebekaburnett** [comment]
@jonkjenn I moved the login-specific things out from her and back to login faq: https://github.com/vippsas/vipps-login-api/pull/225

**Pull request merged**